server/src/main/java/fi/solita/botsofbf/dto/Item.java

```diff
@@ -4,6 +4,8 @@
 
 public class Item {
 
+    public static final int POTION_HEALTH_RESTORE_AMOUNT = 20;
```

Comment:
Näitä lukuja (myös imettävän healthin määrä) voisi testailla vielä, että onko ne miten hyvin tasapainossa.